src/libraries/Microsoft.PowerFx.Core/Public/Logging/ITracer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Logging
+{
+    public interface ITracer
+    {
+        public Task LogAsync(string message, TraceSeverity serv, RecordValue customRecord, CancellationToken ct);
+    }
+
+    public enum TraceSeverity
+    {
+        Information = 0,
+        Warning = 1,
+        Error = 2,
+        Critical = 3
```

Comment:
Those values are different than the ones from Power Apps (see below). Unless we have a very strong reason to change those we should keep them the same so that we can share the enum between multiple platforms.

![Power Apps values](https://i.imgur.com/o8sAq9C.png)

Reply:
That's an interesting expression! Thanks for catching it. Updated.